[2025-04-08T06:06:53.372+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-08T06:06:53.390+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_ingestion_dag.ingest scheduled__2025-04-08T05:00:00+00:00 [queued]>
[2025-04-08T06:06:53.398+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_ingestion_dag.ingest scheduled__2025-04-08T05:00:00+00:00 [queued]>
[2025-04-08T06:06:53.398+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-08T06:06:53.408+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): ingest> on 2025-04-08 05:00:00+00:00
[2025-04-08T06:06:53.418+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=158) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-08T06:06:53.420+0000] {standard_task_runner.py:72} INFO - Started process 160 to run task
[2025-04-08T06:06:53.420+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_ingestion_dag', 'ingest', 'scheduled__2025-04-08T05:00:00+00:00', '--job-id', '259', '--raw', '--subdir', 'DAGS_FOLDER/weather_ingestion_dag.py', '--cfg-path', '/tmp/tmp01trppgf']
[2025-04-08T06:06:53.422+0000] {standard_task_runner.py:105} INFO - Job 259: Subtask ingest
[2025-04-08T06:06:53.500+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_ingestion_dag.ingest scheduled__2025-04-08T05:00:00+00:00 [running]> on host 242fb041fbeb
[2025-04-08T06:06:53.579+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='beni' AIRFLOW_CTX_DAG_ID='weather_ingestion_dag' AIRFLOW_CTX_TASK_ID='ingest' AIRFLOW_CTX_EXECUTION_DATE='2025-04-08T05:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-08T05:00:00+00:00'
[2025-04-08T06:06:53.580+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-08T06:06:54.167+0000] {logging_mixin.py:190} INFO - ✅ Saved: gs://gooutside-raw/weather/bamberg/2025-04-08T06:00:00+02:00.json
[2025-04-08T06:06:54.518+0000] {logging_mixin.py:190} INFO - ✅ Saved: gs://gooutside-raw/weather/bamberg/2025-04-08T07:00:00+02:00.json
[2025-04-08T06:06:54.865+0000] {logging_mixin.py:190} INFO - ✅ Saved: gs://gooutside-raw/weather/bamberg/2025-04-08T08:00:00+02:00.json
[2025-04-08T06:06:55.235+0000] {logging_mixin.py:190} INFO - ✅ Saved: gs://gooutside-raw/weather/bamberg/2025-04-08T09:00:00+02:00.json
[2025-04-08T06:06:55.593+0000] {logging_mixin.py:190} INFO - ✅ Saved: gs://gooutside-raw/weather/bamberg/2025-04-08T10:00:00+02:00.json
[2025-04-08T06:06:55.936+0000] {logging_mixin.py:190} INFO - ✅ Saved: gs://gooutside-raw/weather/bamberg/2025-04-08T11:00:00+02:00.json
[2025-04-08T06:06:56.279+0000] {logging_mixin.py:190} INFO - ✅ Saved: gs://gooutside-raw/weather/bamberg/2025-04-08T12:00:00+02:00.json
[2025-04-08T06:06:56.604+0000] {logging_mixin.py:190} INFO - ✅ Saved: gs://gooutside-raw/weather/bamberg/2025-04-08T13:00:00+02:00.json
[2025-04-08T06:06:56.956+0000] {logging_mixin.py:190} INFO - ✅ Saved: gs://gooutside-raw/weather/bamberg/2025-04-08T14:00:00+02:00.json
[2025-04-08T06:06:57.299+0000] {logging_mixin.py:190} INFO - ✅ Saved: gs://gooutside-raw/weather/bamberg/2025-04-08T15:00:00+02:00.json
[2025-04-08T06:06:57.615+0000] {logging_mixin.py:190} INFO - ✅ Saved: gs://gooutside-raw/weather/bamberg/2025-04-08T16:00:00+02:00.json
[2025-04-08T06:06:57.959+0000] {logging_mixin.py:190} INFO - ✅ Saved: gs://gooutside-raw/weather/bamberg/2025-04-08T17:00:00+02:00.json
[2025-04-08T06:08:37.913+0000] {job.py:229} INFO - Heartbeat recovered after 78.89 seconds
[2025-04-08T06:08:43.373+0000] {connectionpool.py:868} WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, '[SSL: UNEXPECTED_EOF_WHILE_READING] EOF occurred in violation of protocol (_ssl.c:1010)'))': /token
[2025-04-08T06:08:43.672+0000] {logging_mixin.py:190} INFO - ✅ Saved: gs://gooutside-raw/weather/bamberg/2025-04-08T18:00:00+02:00.json
[2025-04-08T06:08:43.673+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-08T06:08:43.703+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-08T06:08:43.704+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=weather_ingestion_dag, task_id=ingest, run_id=scheduled__2025-04-08T05:00:00+00:00, execution_date=20250408T050000, start_date=20250408T060653, end_date=20250408T060843
[2025-04-08T06:08:43.754+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-08T06:08:43.767+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-08T06:08:43.768+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
